config/config.php

```diff
@@ -6,45 +6,57 @@
 
 declare(strict_types=1);
 
-$stableReleaseDate = '2025-01-07 15:00';
-$stableVersion = '3.15.3';
-$fileProviderStableReleaseDate = '2025-01-07 15:00';
-$fileProviderStableVersion = '3.15.3';
+//
+// daily
+//
+$dailyReleaseDateLinux = '20250113';
+$dailyReleaseDateWindows = '20250113';
+$dailyReleaseDateMacos = '20250113';
+$dailyUrl = "https://download.nextcloud.com/desktop/daily/";
 
+//
+// beta
+//
 $betaReleaseDate = '2024-09-09 17:00';
 $betaVersionInternal = '3.13.83';
 $betaVersion = '3.14.0-rc3';
+$betaVersionString = 'Nextcloud Client ' . $betaVersion;
+$betaUrl = 'https://github.com/nextcloud-releases/desktop/releases/download/v' . $betaVersion . '/';
 
-$dailyReleaseDateLinux = '20250113';
-$dailyReleaseDateWindows = '20250113';
-$dailyReleaseDateMacos = '20250113';
-$dailyUrl = "https://download.nextcloud.com/desktop/daily/";
+//
+// stable
+//
+$stableReleaseDate = '2025-01-07 15:00';
+$stableVersion = '3.15.3';
+$stableFileProviderReleaseDate = '2025-01-07 15:00';
```

Comment:
this is no longer needed
```suggestion
```
we eventually managed to release all client builds at the same time